action.yaml

```diff
@@ -229,8 +229,9 @@ runs:
         else
             echo "| Sonar Scan | :${{ steps.all_icons.outputs.sonar_scan }}: |" >> $GITHUB_STEP_SUMMARY
         fi
-
-        if [ "${{ github.event_name }}" == "push" ]; then
+             
+        echo "Blackduck icon '${{ steps.all_icons.outputs.blackduck_scan }}'"
```

Comment:
Does this `echo` matters?

Reply:
No, it is not required. I just added for GH job summary testing purpose